src/ring_theory/ideal/basic.lean

```diff
@@ -429,6 +429,13 @@ section comm_ring
 
 namespace ideal
 
+theorem mul_sub_mul_mem {R : Type*} [comm_ring R] {I : ideal R} {a b c d : R}
```

Comment:
Should I be explicit to enable dot notation here?

Reply:
Yes, I think so

Reply:
Done.